ThriftLibrary.cmake

```diff
@@ -166,13 +166,17 @@ endmacro()
 
 #
 # thrift_generate
+# Supports libary names that differ from the file name (two handle two libraries with the same filename on disk (in different folders))
```

Comment:
"two handle" -> "to handle" and please break the long line for readability here and below.